x-pack/solutions/security/plugins/cloud_security_posture/public/components/accounts_evaluated_widget.tsx

```diff
@@ -63,15 +65,27 @@ export const AccountsEvaluatedWidget = ({
 
   const navToFindingsByCloudProvider = (provider: string) => {
     navToFindings(
-      { 'cloud.provider': provider, 'rule.benchmark.posture_type': CSPM_POLICY_TEMPLATE },
+      activeNamespace
+        ? {
+            [`${FINDINGS_GROUPING_OPTIONS.NAMESPACE}`]: activeNamespace,
```

Comment:
I think it is better not to use the grouping const not for the grouping but just as a general const of the datastream namespace field, it's confusing and couple unrelated features. Maybe update here and in main? I think we have another const that contains just fields of the finding document

Reply:
i'd vote for changing it not to rely on the grouping const for filtering